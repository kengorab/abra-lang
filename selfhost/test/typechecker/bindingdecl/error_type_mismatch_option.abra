val a: Int? = Option.Some("abc")