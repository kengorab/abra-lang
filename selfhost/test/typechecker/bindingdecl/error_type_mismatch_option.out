Error at %FILE_NAME%:1:27
Type mismatch for parameter 'value'
  |  val a: Int? = Option.Some("abc")
                               ^
Expected: Int
but instead found: String
